packages/i18n/docs/ilib.md

```diff
@@ -80,7 +80,7 @@ locale-specific files will typically include values for "title", "keywords", and
 "description".  The other properties will keep the values inherited from the
 app's top-level `appinfo.json`.
 
-`$L()` is a convience method for `ResBundle`, it is described in the main section of the `i18n` docs [here](../index.md)
+`$L()` is a convience method for `ResBundle`, it is described in the main section of the `i18n` docs [here](./index.md)
```

Comment:
Aha!  Yes, this is more correct, though the other works in generated docs.  This works on GitHub!

Reply:
Oh yes, My initial intention was a just change to file location. I miss types as the wrong file name. Following your suggestion, I've given a more specific link. and Also change the base branch. Thank you!